pkg/swarm/swarm.go

```diff
@@ -119,6 +119,15 @@ func (a Address) MarshalJSON() ([]byte, error) {
 	return json.Marshal(a.String())
 }
 
+// Closer returns if x is closer to a than y
+func (x Address) Closer(a Address, y Address) bool {
```

Comment:
I think that if you add this function, it would be good to sweep through the entire codebase and apply this, instead of just in one place